### PR TITLE
correct changelog formatting

### PR DESCRIPTION
**Pull request by glitchcrab**
looks like i missed this and it's causing the release workflow to fail.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -82,14 +82,15 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
-[v1.0.8]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.8
-[v1.0.7]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7
-[v1.0.6]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.6
-[v1.0.5]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.5
-[v1.0.4]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.4
-[v1.0.3]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.3
-[v1.0.2]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.2
-[v1.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.1
+[Unreleased]: https://github.com/giantswarm/cert-manager-app/compare/v2.0.1...HEAD
+[v2.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v2.0.0...v2.0.1
```
Comment (sslavic) [inline CHANGELOG.md:86]:
```suggestion
[Unreleased]: https://github.com/giantswarm/cert-manager-app/compare/v2.0.0...HEAD
```

There's no 2.0.1 yet https://github.com/giantswarm/cert-manager-app/releases

Comment (glitchcrab) [reply CHANGELOG.md:86]: is this more architect magic then? it updates that itself?

File: CHANGELOG.md [hunk 2]
```diff
@@ -82,14 +82,15 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
+[v2.0.0]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.8...v2.0.0
+[v1.0.8]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7...v1.0.8
+[v1.0.7]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.6...v1.0.7
+[v1.0.6]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.5...v1.0.6
+[v1.0.5]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.4...v1.0.5
+[v1.0.4]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.3...v1.0.4
+[v1.0.3]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.2...v1.0.3
+[v1.0.2]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.1...v1.0.2
+[v1.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.0...v1.0.1
 [v1.0.0]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.0
```
Comment (sslavic) [inline CHANGELOG.md:95]:
```suggestion
[2.0.0]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.8...v2.0.0
[1.0.8]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7...v1.0.8
[1.0.7]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.6...v1.0.7
[1.0.6]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.5...v1.0.6
[1.0.5]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.4...v1.0.5
[1.0.4]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.3...v1.0.4
[1.0.3]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.2...v1.0.3
[1.0.2]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.1...v1.0.2
[1.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.0...v1.0.1
[1.0.0]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.0
```

If I recall well, it doesn't like v prefix in the links. Please update the link refs.

File: CHANGELOG.md (hunk 1 above)
Comment (sslavic) [reply CHANGELOG.md]: Yes, it will replace unreleased as part of the (pre-)release workflow

**Pull request merged**